src/EditorFeatures/CSharpTest/Classification/SyntacticTaggerTests.cs

```diff
@@ -33,7 +38,14 @@ public class SyntacticTaggerTests
                     workspace.GetService<IForegroundNotificationService>(),
                     AggregateAsynchronousOperationListener.CreateEmptyListener(),
                     null,
-                    new SyntacticClassificationTaggerProvider(null, null, null));
+                    new SyntacticClassificationTaggerProvider(
+                        null, 
```

Comment:
I can name all the empty & null things being passed in for test purposes if you really want, but it didn't seem necessary.